content/en/events/kcc2020/board-games.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Board Games"
+weight: 1
+slug: "board-games"
+type: docs
+---
+
+Just because we are internet-partying doesn’t mean we need to skimp on the board games! This is not an exhaustive list and if you have any questions please join us in the board-games channel on Discord!
+
+## Web
+
+All of these only require a web browser to get started.
+
+### Board Game Arena
+
+URL: https://boardgamearena.com/
```

Comment:
These aren't showing up as bullets/linefeeds in the markdown preview so they're all on the same line. Same for all the other games.